src/pe_reports/report_generator.py

```diff
@@ -510,29 +538,26 @@ def main():
     logging.basicConfig(
         format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
     )
-
-    # TODO: Add generate_reports func to handle cmd line arguments and function.
-    # Issue #8: https://github.com/cisagov/pe-reports/issues/8
-    # def generate_reports(db, datestring, data_directory, output_directory):
-
     logging.info(
         "Loading Posture & Exposure Report Template, Version : %s", __version__
     )
     logging.info("Generating Graphs")
 
     # Create output directory
-    subprocess.call(["mkdir", args["OUTPUT_DIRECTORY"]])
+    try:
+        os.mkdir(f"{args['OUTPUT_DIRECTORY']}")
+    except FileExistsError as err:
+        logging.error(f"The output directory cannot be created. {err}")
+        return 0
```

Comment:
Same question about existing output directories with the addition that the error message does not accurately describe the exception.

Reply:
@mcdonnnj , 

I have update the code to better describe the error state per commit [0bb568a](https://github.com/cisagov/pe-reports/pull/33/commits/0bb568a663eb003c82d860ba857afaf1c140f77d)

Reply:
Those changes need to be made here as well. This logging message still does not accurately reflect the exception you are catching and it should return `1` to indicate an error.